101-javaee-like-getting-started/src/main/java/io/quarkus/qe/core/GreetingHealthCheck.java

```diff
@@ -1,12 +1,13 @@
 package io.quarkus.qe.core;
 
-import org.eclipse.microprofile.health.Health;
+import javax.enterprise.context.ApplicationScoped;
+
 import org.eclipse.microprofile.health.HealthCheck;
 import org.eclipse.microprofile.health.HealthCheckResponse;
 
-import javax.enterprise.context.ApplicationScoped;
+import io.smallrye.health.api.HealthGroup;
 
-@Health
+@HealthGroup("greeting")
```

Comment:
Why SRye specific annotation and not the MicroProfile one?

Reply:
There is no something similar in MicroProfile, but in SRye. 
Another option is to cover the annotations @Liveness and @Readiness, but this functionality is well covered in upstream already. 
What do you think?

Reply:
I've changed the `@Health` to `@Readiness` and added a new test case to cover `@HealthGroup`.